src/types/right-holder-type.ts

```diff
@@ -0,0 +1,28 @@
+import { EosName } from '../smart-contract-types/base-types';
+import { ExecutionType, ApprovalType } from './smart-contract-enum';
+
+export interface RightHolderType {
+  /**
+   * This param is required if the code is COLLECTIVE_DECISION execution type
+   */
+  exec_type?: ExecutionType;
+  approval_type?: ApprovalType;
+  sole_right_accounts?: [EosName];
```

Comment:
@danielAlvess could you change to `EosName[]`, just to keep consistency.

Reply:
Btw, if you think param is hard to understand, please add document then.

Reply:
updated